test/src/page_models/access-user-card.js

```diff
@@ -8,6 +8,13 @@ class AccessUserCard {
         this.deleteEdxUserConfirmationDialog = Selector(`#edxUser-${edxUserID} .deleteEdxUserConfirmationDialog`);
         this.cancelUserDeleteButton = Selector(`#edxUser-${edxUserID} .cancelUserDeleteButton`);
         this.confirmUserDeleteButton = Selector(`#edxUser-${edxUserID} .confirmUserDeleteButton`);
+        this.schoolSearchSection = Selector('div').withText('Search a school below to manage their EDX Access');
+        this.selectSchoolDropdown = Selector('#selectInstituteName');
+        this.manageSchoolButton = Selector('#manageSchoolButton');
+        this.editEdxUserButton =  Selector(`#edxUser-${edxUserID} .editEdxUserButton`);
```

Comment:
I think the selectors for the edit and remove buttons use a mix of last and first name to generate. (Which has it's own faults since white spaces aren't allowed in id's). 

We can change this to put in the edxUserID.